src/status_im/contexts/wallet/effects.cljs

```diff
@@ -13,6 +13,11 @@
   [kw]
   (-> kw symbol str))
 
+(rf/reg-fx
+ :effects.wallet/get-random-mnemonic
+ (fn [{:keys [on-success]}]
+   (native-module/get-random-mnemonic on-success)))
+
```

Comment:
Don't we already have an effect to perform this call? :eyes:

Reply:
This effect was removed during refactoring